apps/backend/README.md

```diff
@@ -0,0 +1,37 @@
+# Backend
+
+Uses Supabase. You need docker running locally to start local dev. You only need to do that when editing.
+
+## Handy scripts:
+
+Note: all scripts assume the backend directory is the current working directory.
+
+Generating migrations when you've edited the DB in the console.
+
+```
+yarn supabase db diff --use-migra name_of_migration -f name_of_migration
+```
+
+Reset db after migration to test migration files
```

Comment:
todo: can you document what migration files are in this context or provide an outbound documentation link please?